security/official-domains.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Official Domains
+description: Domains owned by MyCrypto
+priority: 100
+date_published: 2019-02-27
+date_modified: 2019-02-27
+---
+
+MyCrypto uses several domains to run the service. Each domain serves a specific purpose, such as; running the interface, running the support center, running a public staging area, running RPC nodes.
+
+Check the domains in this article if you're unsure if it's safe to go to a particular website or accept a communication from.
+
+## mycrypto.com
+
+Most of the things a typical user is interested on will be on the `mycrypto.com` domain - including various subdomains.
+
+### Verified mycrypto.com domains
+
+All MyCrypto content should originate from one of the following domains;
+
+* `mycrypto.com` - the production-stable blockchain interface
+* `about.mycrytpo.com` - team profile and job opportunities
+* `beta.mycrypto.com` - a public staging area
+* `buy.mycrypto.com` - a portal to buy Ethereum and Bitcoin with your credit card
+* `download.mycrypto.com` - download our desktop app
+* `legacy.mycrypto.com` - the old interface look
+* `overflow.mycrypto.com` - a place to download developer tools
+* `support.mycrypto.com` - our knowledge base
+* `winter.mycrypto.com` - our crypto winter calendar
+* `www.mycrypto.com` - the production-stable blockchain interface
+* `0x.mycrypto.com` - holding the 0x protocol code used by the MyCrypto interface
+
+## mycryptoapi.com
+
+This domain holds many background processes that we rely on.
+
+### Verified mycryptoapi.com domains
+
+* `proxy.mycryptoapi.com` - handles proxying requests to third-parties in the background
+* `raidentransport.mycryptoapi.com` - [matrix] server for Raiden related stuff
+* `api.mycryptoapi.com` - handles requests to our Blockchain RPC nodes
+* `apis.mycryptoapi.com` - used for weighted load balancing between two regions
+
+## mycryptobuilds.com
+
+This domain is for staging and QA to test builds for our blockchain interface. Since the code 
+running the interface is in test phase, there may be bugs - don't use it with the same
+conviction as production (`mycrypto.com`).
+
+### Verified mycryptobuilds.com domains
+
+* `mycryptobuilds.com` - serves QA builds for the blockchain interface
+
+### Verified mycryptoapi.com domains
+
+* `proxy.mycryptoapi.com` - handles proxying requests to third-parties in the background
+* `raidentransport.mycryptoapi.com` - [matrix] server for Raiden related stuff
+* `api.mycryptoapi.com` - handles requests to our Blockchain RPC nodes
+* `apis.mycryptoapi.com` - used for weighted load balancing between two regions
+
+
+## Official email from MyCrypto
+
+If you recieve a communication that looks like it's from MyCrypto but _doesn't_ come from

```

Comment:
`_doesn't_` -> `*doesn't*`

Reply:
Updated in https://github.com/MyCryptoHQ/knowledge-base-content/pull/33/commits/c2d81736ceb6c173e191cc793222deda4e72b4c0